### PR TITLE
Issue #37 Add a Subscribe page 

### PR DESCRIPTION
**Pull request by shirmenzely**
By Ron Biton and Shir Menzely

### CODE REVIEW

File: Subscribe page.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from flask import Flask, request, render_template
```
Comment (lmilbaum) [inline Subscribe page.py:1]: The subscribe page should be and addition to the current app, not another app with a subscribe page

**Review by lmilbaum — CHANGES_REQUESTED** [review]
Go over all the code review comments provided today. And find out what is relevant for this PR.

**Comment by lmilbaum** [comment]
@shirmenzely we have a new contact_us page. Have a look.  It might be a duplicate of this work.

**Comment by shirmenzely** [comment]
> @shirmenzely we have a new contact_us page. Have a look. It might be a duplicate of this work.

I think the essence of the pages is different. On the contact page users leave a message, while on the second page they subscribe for updates

**Comment by lmilbaum** [comment]
Is this PR still relevant or we can close it?

**Pull request closed without merging**